calico_node/tests/ut/test_log_parsing.py

```diff
@@ -41,7 +41,7 @@
 2017-01-12 19:19:04.419 [INFO][87] int_dataplane.go 389: Received interface update msg=&intdataplane.ifaceUpdate{Name:"lo", State:"up"}
 2017-01-12 19:19:04.419 [INFO][87] ipip_mgr.go 95: Removing old address addr=192.168.151.0 link="tunl0" oldAddr=192.168.151.0/32 tunl0
 2017-01-12 19:19:04.416 [INFO][87] syncer.go 247: etcd watch thread started.
-2017-01-12 19:19:04.419 [INFO][87] int_dataplane.go 378: Received update from calculation graph msg=config:<key:"ClusterGUID" value:"328b9309c07447db893cd7c155f2547b" > config:<key:"DefaultEndpointToHostAction" value:"RETURN" > config:<key:"EtcdAddr" value:"" > config:<key:"EtcdCaFile" value:"" > config:<key:"EtcdCertFile" value:"" > config:<key:"EtcdEndpoints" value:"http://10.96.232.136:6666" > config:<key:"EtcdKeyFile" value:"" > config:<key:"EtcdScheme" value:"" > config:<key:"FelixHostname" value:"tigera-lwr-kubetest-02" > config:<key:"InterfacePrefix" value:"cali" > config:<key:"IpInIpEnabled" value:"true" > config:<key:"IpInIpTunnelAddr" value:"192.168.151.0" > config:<key:"IpfixCollectorAddr" value:"192.168.67.75" > config:<key:"IpfixCollectorPort" value:"4739" > config:<key:"LogFilePath" value:"None" > config:<key:"LogSeverityFile" value:"None" > config:<key:"LogSeverityScreen" value:"info" > config:<key:"MetadataAddr" value:"None" > config:<key:"ReportingIntervalSecs" value:"0" > config:<key:"marker" value:"created" >
+2017-01-12 19:19:04.419 [INFO][87] int_dataplane.go 378: Received update from calculation graph msg=config:<key:"ClusterGUID" value:"328b9309c07447db893cd7c155f2547b" > config:<key:"DefaultEndpointToHostAction" value:"RETURN" > config:<key:"EtcdAddr" value:"" > config:<key:"EtcdCaFile" value:"" > config:<key:"EtcdCertFile" value:"" > config:<key:"EtcdEndpoints" value:"http://10.96.232.136:6666" > config:<key:"EtcdKeyFile" value:"" > config:<key:"EtcdScheme" value:"" > config:<key:"FelixHostname" value:"tigera-lwr-kubetest-02" > config:<key:"InterfacePrefix" value:"cali" > config:<key:"IpInIpEnabled" value:"true" > config:<key:"IpInIpTunnelAddr" value:"192.168.151.0" > config:<key:"IpfixCollectorAddr" value:"192.168.67.75" > config:<key:"IpfixCollectorPort" value:"4739" > config:<key:"LogFilePath" value:"None" > config:<key:"LogSeverityFile" value:"None" > config:<key:"LogSeverityScreen" value:"info" > config:<key:"MetadataAddr" value:"None" > config:<key:"ReportingInterval" value:"0" > config:<key:"marker" value:"created" >
```

Comment:
do we need this to change?  I can't actually see what has changed.

Reply:
`ReportingIntervalSecs` -> `ReportingInterval`